include/tiramisu/core.h

```diff
@@ -1144,13 +1188,21 @@ class buffer
       * the common case), the function that was created automatically
       * during Tiramisu initialization will be used (we call that
       * function the "implicit function").
+      * 
+      * \p corr is the name of the cpu buffer corresponding to a gpu buffer. 
+      * This field is only set, when we creat a gpu buffer.
+      *
+      * \p tag is char to spécify in wich memory the gpu buffer will be stored in, 
+      *  It can either take the default value 'g' for global memory, or 'c' to tag 
+      *  the constant memory.
       *
       * Buffer names should not start with _ (an underscore).
       * Names starting with _ are reserved names.
       */
     buffer(std::string name, std::vector<tiramisu::expr> dim_sizes,
            tiramisu::primitive_t type, tiramisu::argument_t argt,
-	   tiramisu::function *fct = global::get_implicit_function());
+           tiramisu::function *fct = global::get_implicit_function(), std::string corr= "",char tag='g');
```

Comment:
Coding style: space before/after "=". Please check our current style which is in the same line.

Reply:
I don't agree with passing tag here. It should be retrieved automatically by the constructor of buffer.